api/src/main/java/org/openmrs/module/ugandaemrreports/reporting/dataset/definition/SharedDataDefintion.java

```diff
@@ -36,4 +41,29 @@ public DataDefinition definition(String name, Concept concept) {
         obsForPersonDataDefinition.setWhich(TimeQualifier.LAST);
         return obsForPersonDataDefinition;
     }
+    public DataDefinition villageParish(){
```

Comment:
@solemabrothers move to a public data definition

Reply:
@ssmusoke I don't get what you mean by public data definition as the file SharedDataDefinitiion is equally public and shared across registers

Reply:
@solemabrothers sorry must have gotten lost in the code - Resolving this